lib/media/media_source_engine.js

```diff
@@ -312,7 +312,7 @@ shaka.media.MediaSourceEngine = class {
       'audio/mp4; codecs="mp4a.40.2"',
       'audio/mp4; codecs="ac-3"',
       'audio/mp4; codecs="ec-3"',
-      'audio/mp4; codecs="ac-4"',
+      'audio/mp4; codecs="ac-4.02.01.01"',
```

Comment:
What's the reason you went with this specific codecs string? The doc lists multiple different codecs strings. It looks like all of them have a `bitstream_version` of 2 and a `presentation_version` of 1, but they show `mdcompat` values ranging from 0 to 3.

I'm not familiar with what the compatibility level represents... is 1 perhaps the most commonly used value in practice?

Reply:
I used a Dolby-generated stream to update this value; see https://webapi.streaming.dolby.com/v0_9/sources/media/v01/dash/lesson_8.mpd

Reply:
Fair enough.